ext/afform/admin/info.xml

```diff
@@ -28,6 +28,7 @@
   </civix>
   <classloader>
     <psr4 prefix="Civi\" path="Civi"/>
+    <psr0 prefix="CRM_" path="."/>
```

Comment:
shouldn't the path be CRM?

Reply:
@seamuslee001 no - the above was the conclusion reached between @demeritcowboy & @totten  & added to Civix

Reply:
![image](https://user-images.githubusercontent.com/336308/195050713-671e6913-baec-4503-b8da-bf0405bb28dd.png)

Reply:
What's the difference between `""` and `"."`?

Reply:
yeah - that was something that @totten & @demeritcowboy discussed & agreed during the training - I don't know but it is the combined wisdom of 2 very large brains to add the version I added to civix

Reply:
Where I got "." from was [this issue](https://github.com/civicrm/civicrm-core/pull/24580#issuecomment-1256406414) where I first tried "" and it didn't work and then I banged on the keyboard until I got the page to load. Isn't that how everyone does civi development?

It could be specific to the class-scanner and the extension and something about what it's looking for that might explain why the ones with "" aren't a problem. Maybe the csvimporter extension does something in a CRM class that others don't.